src/HttpClient/Plugin/Authentication/NullAuthentication.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
Yeah, it had 2019 because originally I had both classes, but now that we've removed the one in the tests, it makes sense that it is the same file that was moved.